src/main/java/org/springframework/integration/aws/inbound/kinesis/KclMessageDrivenChannelAdapter.java

```diff
@@ -84,6 +85,23 @@
 
 	private static final ThreadLocal<AttributeAccessor> attributesHolder = new ThreadLocal<>();
 
+	/**
+	 * Interval to run lease cleanup thread in {@link LeaseCleanupManager}.
+	 */
+	private static final long DEFAULT_LEASE_CLEANUP_INTERVAL_MILLIS = Duration.ofMinutes(1).toMillis();
+
+	/**
+	 * Threshold in millis at which to check if there are any completed leases (leases for shards which have been
+	 * closed as a result of a resharding operation) that need to be cleaned up.
+	 */
+	private static final long DEFAULT_COMPLETED_LEASE_CLEANUP_THRESHOLD_MILLIS = Duration.ofMinutes(5).toMillis();
+
+	/**
+	 * Threshold in millis at which to check if there are any garbage leases (leases for shards which no longer exist
+	 * in the stream) that need to be cleaned up.
+	 */
+	private static final long DEFAULT_GARBAGE_LEASE_CLEANUP_THRESHOLD_MILLIS = Duration.ofMinutes(30).toMillis();
```

Comment:
I wonder what was the reason for AWS to keep exatly these constants in the `KinesisClientLibConfiguration` as `private`...

Anyway: LGTM - merging...